dev-tools/get-bwc-version.py

```diff
@@ -65,7 +65,7 @@ def main():
     # 1.2.0 was pulled from download.elasticsearch.org because of routing bug:
     url = 'http://central.maven.org/maven2/org/elasticsearch/elasticsearch/1.2.0/%s' % filename
   else:
```

Comment:
this won't allow us to downlaod 1.x version and before shoudl we do the same thing here like:
```diff
-    url = 'https://download.elasticsearch.org/elasticsearch/elasticsearch/%s' % filename
+    if c.version.startswith('0') or c.version.startswith('1'):
+      url = 'https://download.elasticsearch.org/elasticsearch/elasticsearch/%s' % filename
+    else:
+      url = 'http://download.elasticsearch.org/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/%s/%s' % (c.version, filename)
+
```